draft-ietf-ppm-dap.md

```diff
@@ -2701,7 +2701,8 @@ encoded(struct { prepare_resps } AggregationJobResp)
 
 If for whatever reason, the Leader must abandon an aggregation job, it SHOULD
 let the Helper know it can clean up its state by sending a DELETE request to the
-job.
+job. Deletion of an aggregation job MUST NOT delete information needed for
+replay or double collection checks ({{replay-protection}}).
```

Comment:
I wonder if {{batch-buckets}} would be a better reference?

Reply:
It's counter intuitive but you're right, {{batch-buckets}} has more of the fine details of how to enforce replay and double collection.